jenkins/debian/debos/stretch.yaml

```diff
@@ -0,0 +1,88 @@
+{{- $architecture := or .architecture "arm64" -}}
+{{- $basename := or .basename "." -}}
+{{- $extra_packages := or .extra_packages "" -}}
+
+architecture: {{ $architecture }}
+
+actions:
+  - action: debootstrap
+    suite: stretch
+    components:
+      - main
+    mirror: http://deb.debian.org/debian
+    variant: minbase
+
+  - action: apt
+    recommends: false
+    packages:
+      - udev
+      - kmod
+      - dbus
+      - systemd-sysv
+      - diffutils
+      - usbutils
+
+  - action: run
+    description: Install extra packages
+    chroot: true
+    command: apt-get update && apt-get -y install --no-install-recommends {{ $extra_packages }}
+
+  - action: run
+    description: Set hostname
+    chroot: true
+    command: echo debian > /etc/hostname
```

Comment:
Could you add $architecture to the hostname?  Ideally even the debian distro name.  e.g. "arm64-debian-stretch", or just "arm64-stretch".